### ISSUE TITLE
feat(docs): Add Versioning to Docs

### ISSUE THREAD

**Issue by flxwu** [langfuse/langfuse#711]
### Describe the feature you'd like to request

<img width="1434" alt="image" src="https://github.com/langfuse/langfuse/assets/16862997/b890d167-3800-4990-bfdb-19fe3beca2cc">

### Describe the solution you'd like to see

Nextra doesn't support this out-of-the-box yet: https://github.com/shuding/nextra/issues/20 :/ 

(note: Docusaurus and some others do - just as note here, i'm not proposing to migrate :))

### Contribute

- [ ] Yes, I can implement this and raise a PR

**Comment by marcklingen** [comment]
I agree, having versioning for docs would be helpful, especially considering the upcoming small breaking change in the v2 SDKs. However, migrating away from Nextra is currently too big of a lift. Hope this will soon be included in Nextra.

**Comment by sleeporsmile** [comment]
Why it is low priority? it's joke?
